website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,192 @@
+| `-k8-context`          | String value that specifies the Kubernetes context to use when starting the Consul server.                                                                                            | Optional | current context                                                         |
+| `-k8-namespace`        | String value that specifies the Kubernetes namespace to use when starting the Consul server.                                                                                          | Optional | `default`                                                               |
+| `-log-json`            | Boolean value that enables or disables JSON format for the log output.                                                                                                                | Required | `false`                                                                 |
+| `-log-level`           | String value that specifies the logging level. The following values are supported: <br/>- `trace` (highest level of detail) <br/>- `debug` <br/>- `info` <br/>- `warn` <br/>- `error` | Optional | `info`                                                                  |
+| `-metrics-port`        | Integer value that specifies the port number for collecting metrics.                                                                                                                  | Optional | none                                                                    |
+| `-pprof`               | Integer value that specifies the Go pprof port number for collecting metrics.                                                                                                         | Optional | none                                                                    |
+| `-sds-server-host`     | String value that specifies the host server for the secret discovery service (SDS).                                                                                                   | Optional | `consul-api-gateway-controller.default.`<br/>`svc.cluster.`<br/>`local` |
+| `-sds-server-host`     | Integer value that specifies the port number for the secret discovery service (SDS).                                                                                                  | Optional | `9090`                                                                  |
+
+You can also issue the `version` command to print the Consul API Gateway version to the console:
+
+```shell-session
+$ ./consul-api-gateway version
+consul-api-gateway 0.1.0
+```
+
+## Configuration
+
+Create the following artifacts to configure the API Gateway:
+
+- [Gateway](#gateway): Defines the gateway properties, including listeners.
+- [Listeners](#listeners): Defines listener properties, such as protocol, port, and namespace.
+- [Routes](#routes): Specifies the path from the client to the listener.
+
+### Gateway
```

Comment:
We should add a section on `GatewayClass`/`GatewayClassConfig` here -- the installation process will install some default stuff, but the `GatewayClassConfig` is implementation-specific, so if not here it won't be documented anywhere else.